### PR TITLE
week1 example implementation

### PR DESCRIPTION
**Pull request by madhupriyaallam**
Implemented week1 example (i.e) Home Automation

### CODE REVIEW

File: src/week1/WaterMotor.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+public class WaterMotor implements WaterMotorAutomation{
+    private double topLevel;
+    WaterMotor(double topLevel){
+        //this.bottomLevel=bottomLevel;
+        this.topLevel=topLevel;
+    }
+    @Override
+    public void automateWaterMotor(double level) {
+        if (level == topLevel){
+            System.out.println("Sensor identifies the bottom level is reached and Water-motor gets off");
```
Comment (Hemant-GH) [inline src/week1/WaterMotor.java]: off when reaches top level not bottom level

File: src/week1/WaterMotorAutomation.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package week1;
+
+public interface WaterMotorAutomation {
+    public void automateWaterMotor(double level);
```
Comment (Hemant-GH) [inline src/week1/WaterMotorAutomation.java]: method should be automate. interface name is clear to identify that water motor is being automated

File: src/week1/ElectricAppliancesAutomation.java [hunk 3]
```diff
@@ -0,0 +1,5 @@
+package week1;
+
+public interface ElectricAppliancesAutomation {
+    public void automateElectricAppliances(boolean sensorMsg);
```
Comment (Hemant-GH) [inline src/week1/ElectricAppliancesAutomation.java]: method name should be automate. Create Top interface Automation which would then be extended by ElectricAppiance and WaterMotor

Comment (madhupriyaallam) [reply src/week1/ElectricAppliancesAutomation.java]: Hemant, here method signature is different for both the methods I can't change their name or can't create top level interface for this

**Review by Hemant-GH — APPROVED** [review]
Never commit folders like idea and out in repo

**Comment by madhupriyaallam** [comment]
> 

Okay hemant, will do that from now

**Pull request merged**